quarto/03-QC-LBT01.qmd

```diff
@@ -0,0 +1,172 @@
+---
+title: "QCing Laboratory Test Results and Change from Baseline by Visit (LBT01)"
+editor: source
+format: 
+  html:
+    page-layout: full
+    code-fold: true
+    code-summary: "Show the code"
+    code-overflow: scroll 
+    df-print: kable
+---
+
+## Laboratory Test Results and Change from Baseline by Visit (LBT01) QC Workflow
+
+### 1. Generate a table using {chevron}
+
+Using the `lbt01_main()` function from the {chevron} package to generate a Lab Results Table. We'll add an overall column to walk through the QCing steps if your table has one.
+
+```{r}
+#| message: false
+#| code-summary: "Show the code"
+library(chevron)
+
+# Create a table using the chevron package 
+tlg_lbt01 <- chevron::lbt01_main(syn_data, lbl_overall = "Overall ARM")
+head(tlg_lbt01, n = 15) #truncated display
```

Comment:
```suggestion
head(tlg_lbt01, n = 15)
```
I think head is understandable (I also add this sometimes but I am thinking could be redundant)